src/main/java/com/clickhouse/kafka/connect/sink/kafka/RangeContainer.java

```diff
@@ -66,6 +66,9 @@ public RangeState getOverLappingState(RangeContainer rangeContainer) {
         // ZEROED [10, 20] Actual [0, 10]
         if (actualMinOffset == 0)
             return RangeState.ZERO;
+        // PREVIOUS
```

Comment:
Please provide an example like `// ZEROED [10, 20] Actual [0, 10]` with the correct values